tool/flutter_ci_script_master.sh

```diff
@@ -25,6 +25,7 @@ declare -ar PROJECT_NAMES=(
     "compass_app/app"
     "compass_app/server"
     "context_menus"
+    "date_planner"
```

Comment:
Given this has `sdk: ^3.5.0` this should be added to `master`, `beta`, and `stable`.